components/brave_rewards/core/publisher/publisher.cc

```diff
@@ -27,6 +27,30 @@
 #include "brave/components/brave_rewards/core/state/state.h"
 
 namespace brave_rewards::internal {
+
+namespace {
+
+bool IsPublisherExcluded(mojom::PublisherInfo& publisher_info) {
+  if (publisher_info.excluded == mojom::PublisherExclude::EXCLUDED) {
+    return true;
+  }
+  switch (publisher_info.status) {
+    case mojom::PublisherStatus::NOT_VERIFIED:
+      return true;
+    case mojom::PublisherStatus::UPHOLD_VERIFIED:
+      break;
+    case mojom::PublisherStatus::BITFLYER_VERIFIED:
+      break;
+    case mojom::PublisherStatus::GEMINI_VERIFIED:
+      break;
```

Comment:
Nit: Could be condensed to drop-through and return false, unless we're trying to be extra explicit here. Not a big deal either way.

Reply:
Easier on the eyes - thanks!